lib/models/item.dart

```diff
@@ -90,6 +90,9 @@ class Item with _$Item {
   late final bool flaggable =
       active && type != ItemType.job && type != ItemType.pollopt;
 
+  late final bool visitable =
+      active && type == ItemType.story;
```

Comment:
The `job` and `poll` `ItemType`s should also be visitable, so I'd use something like this, similar to other entries:

```suggestion
      active && type != ItemType.job && type != ItemType.pollopt;
```

Reply:
btw isn't it the other way around here? In this case visitable is true if it is not a job and also not a poll...

Reply:
Oh, I did mess up the suggestion. It should've been:

```dart
      active && type != ItemType.comment && type != ItemType.pollopt;
```

There's a difference between `poll` and `pollopt`. The former behaves like a story and can be opened from the stories list, while the latter is one of the choices within a poll. For instance, https://news.ycombinator.com/item?id=21231805, which is not really visitable.

Reply:
Got it and fixed it!